torch/_dynamo/output_graph.py

```diff
@@ -1021,6 +1022,12 @@ def call_user_compiler(self, gm: fx.GraphModule) -> CompiledFn:
             )
             unimplemented_with_warning(e, self.root_tx.f_code, msg)
         except Exception as e:
+            if (
+                isinstance(e, RuntimeError)
+                and e.args[0] == "There is unbacked SymInt in the graph"
```

Comment:
It would still be much better to avoid testing against a specific error string here

Reply:
Done